src/ts/vttutils.ts

```diff
@@ -128,7 +128,7 @@ const setVttPositionAlign = (cueContainerDom: DOM, vtt: VTTProperties, direction
         break;
       case 'line-right':
         cueContainerDom.css(direction, 'auto');
-        cueContainerDom.css(DirectionPair.get(direction), `${vtt.position}%`);
+        cueContainerDom.css(DirectionPair.get(direction), `${100 - vtt.position}%`);
```

Comment:
This basically turns the `line-right` into `line-left` as far as i can tell.
If I use `position:0%,line-left` the content will be aligned on the left border (0% from flex-start)
if I use `position:0%,line-right` the content will also be aligned on the left border since it will get converted to 100% from flex-end

this can't be right, can it?

Reply:
Yes, this is correct and the content should at the left border when you use **position: 0%** because it is **0% in the writing direction**, that is from the left side of the container.

The `line-left` and `line-right` does **not define the offset of the text/cue box**, rather in which direction the box will grow or where to put the text so that it will match the offset provided by position. However you want to understand.